includes/admin/views/tools/danger-zone.php

```diff
@@ -0,0 +1,57 @@
+<div class="erp-grid-container tools-section erp-danger-zone">
+    <h3 class="section-title"><?php esc_html_e('Reset WP ERP', 'erp'); ?></h3>
```

Comment:
Considering WP coding standard, there should be a space after the opening parentheses and before the closing parentheses while using a function, `i.e., esc_html_e( 'Reset WP ERP', 'erp' )`